tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchLiveIndexingTests.java

```diff
@@ -79,17 +73,17 @@
 
     /**
      * Data will be prepared using the schema below:
-     * 
+     * <p>
      * Site1:
-     *  - Users: userSite1, userMultiSite
-     *  - Documents: FILE_0_NAME (owner: userSite1), FILE_1_NAME (owner: userSite1), FILE_3_NAME (owner: userSite2)
-     *  
+     * - Users: userSite1, userMultiSite
+     * - Documents: FILE_0_NAME (owner: userSite1), FILE_1_NAME (owner: userSite1), FILE_3_NAME (owner: userSite2)
+     * <p>
```

Comment:
[Minor] These paragraph markers are right, but I think the list should really be in an `<ul>` tag each with an `<li>`?  I doubt anyone's actually going to read the generated javadoc though for this - I suppose they're much more likely to look at the code.